src/components/Home/WhatsNewModal/styles.module.css

```diff
@@ -0,0 +1,155 @@
+.container {
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 11;
+
+  &:hover {
+    cursor: pointer;
+  }
+
+  &Open {
+    height: 100%;
+    width: 100%;
+    background: rgba(132, 142, 148, 0.5);
+  }
+
+  @media (--xs-scr) {
```

Comment:
We don't have tablet or mobile versions of the model yet. For now, I kept the model in the tablet screens and took it off for the mobile screens.